src/test/java/org/jabref/logic/integrity/PersonNamesCheckerTest.java

```diff
@@ -31,4 +51,15 @@ public void complainAboutPersonStringWithTwoManyCommas() throws Exception {
     public void doNotComplainAboutSecondNameInFront() throws Exception {
         assertEquals(Optional.empty(), checker.checkValue("M. J. Gotay"));
     }
+
+    @Test
+    public void validCorporateNameInBrackets() throws Exception {
+        assertEquals(Optional.empty(), checker.checkValue("{JabRef}"));
+    }
+
+    @Test
+    public void validCorporateNameAndPerson() throws Exception {
+        assertEquals(Optional.empty(), checker.checkValue("{JabRef} and Stefan Kolb"));
```

Comment:
Corporate authors at the end and in the middle work then also, right?

Reply:
I guess so. It only depends on the AuthorsList Parser and the BracketRemover. I just added a few tests here to double test the scenario.